docs/features/features.rst

```diff
@@ -21,6 +21,7 @@ It includes:
 - Spatially heterogeneous spectra (see :ref:`see line-of-sight <label_los_index>`)
 - Post-processing tools to load and :ref:`compare with experimental spectra <label_spectrum_howto_compare>` (see :ref:`the Spectrum object <label_spectrum>`)
 - A :ref:`Line Survey <label_spectrum_linesurvey>` tool to identify which lines correspond to a spectral feature.
+- Matlab users can access RADIS easily and use it from Matalb (see `Access RADIS <https://radis.readthedocs.io/en/latest/examples/examples.html>`__)
```

Comment:
Can you add a :ref: syntax here? Adding a _label in the example.

See the other examples and how Sphinx handles references

Reply:
i have done the changes .